ajax/libs/aquarelle/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "aquarelle",
+  "filename": "aquarelle.min.js",
+  "homepage": "https://ramotion.com/gthb",
```

Comment:
I don't think this is lib's homepage, because it seems like company website.
What do you think?
Thanks

PS. https://github.com/Ramotion/aquarelle/blob/master/bower.json#L4